science/2021-07-08-science-blog-2/index_de.md

```diff
@@ -54,7 +54,7 @@ Die ereignisbezogene Befragung (EDUS) hat u.&nbsp;a. ergeben, dass
  - die meisten Befragten (61,6%) Abitur, eine fachgebundene Hochschulreife oder die Fachhochschulreife erreicht haben,
  - die Befragten mehrheitlich im städtischen Raum und im Westen Deutschlands leben,
  - sich die Mehrheit der Befragten mit den klassischen AHA-Regeln vor einer Infektion geschützt haben,
- - sich die große Mehrheit (83,1%) nach einer „roten Warnung“ testen ließ.
+ - sich die große Mehrheit (83,1%) nach einer „roten Warnung“ testen lassen möchte.
```

Comment:
maybe "wollte sich noch testen lassen"?

Reply:
"[...] nach einer „roten Warnung“ testen lassen wollte."? What do you think?

Reply:
@HelTob 
Your suggestion is simpler and correct, so go with it!